blog.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Sai Samant</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="hello">
+        <h1>&lt; fun stuff &gt;</h1>
+      </div>
+      <div class="nav">
+        <nav>
+          <ul>
+            <li><a href="index.html">home</a></li>
+            <li><a href="portfolio.html">code</a></li>
+            <li><a href="about.html">me</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="all-interests">
+        <article class="interest">
+          <article class="podcasts">
```

Comment:
You never seem to use any of these classes ("podcasts", "knitting", "sociolinguistics", etc) in your CSS or elsewhere. What purpose do they serve, and why do each of your interests have their own class?